pandas/tests/frame/test_indexing.py

```diff
@@ -1974,9 +1963,8 @@ def test_get_set_value_no_partial_indexing(self):
         # partial w/ MultiIndex raise exception
         index = MultiIndex.from_tuples([(0, 1), (0, 2), (1, 1), (1, 2)])
         df = DataFrame(index=index, columns=range(4))
```

Comment:
i would most/all of the _get_value and _set_value test away

Reply:
im not sure we have other tests for _get_value/_set_value, which are still used internally

Reply:
Any thought to migrating it to `test_interals` then? I can see the value of separating these out from other tests at the very least